### DISCUSSION TITLE
Able to use two colors in the key

### DISCUSSION THREAD

**Discussion by JorgeAlMoLa** [fastfetch-cli/fastfetch#1024] [Q&A]
## Disclaimer:
I don't know if this feature is already available or not, I couldn't find it implemented in my config.jsonc.

## Feature:
Being able to use more than one color in the key on the same line
```jsonc
{
    "type": "os",
    "key1": "|",
    "key2": " OS",
    "keyColor1": "darkblue",
    "keyColor2": "cyan"
}
```
![ayuda](https://github.com/fastfetch-cli/fastfetch/assets/102344561/eff4938f-8b9d-4bea-8f54-abd425eb636d)

**Comment by CarterLi** [comment]
```json
{
    "type": "os",
    "key": "#{blue}| #{cyan}OS"
}
```

**Comment by JorgeAlMoLa** [comment]
Isn't working
![h](https://github.com/fastfetch-cli/fastfetch/assets/102344561/a53c31bf-a2da-4aeb-b850-e9a382d8a909)

**Comment by CarterLi** [comment]
Should be `{#blue}| {#cyan}OS`

**Comment by CarterLi** [comment]
You may refer to `fastfetch -h format` and `fastfetch -h color`
